cornice_swagger/swagger.py

```diff
@@ -344,36 +347,34 @@ def from_schema(self, schema_node, validators=[]):
         if not isinstance(schema_node, colander.Schema):
             schema_node = schema_node()
 
-        if colander_validator in validators:
```

Comment:
Yes this is intentional. `schema` is descriptive attribute storing the schema for the service (see Cornices/cornice#376), and `colander_validator` is just a shorthand validator that validate the request against the schema defined on the service. The original intention was to to keep schema and validation two different concerns, and I think that swagger should use only the schema because it serves the same precise concern (describing the service).
The introduction of `colander_body_validator` broke that behaviour, schema and validator became tightly coupled, so `colander_body_validator` should be treated as special case.